applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -283,12 +284,26 @@ static int adv_create(void)
 	return 0;
 }
 
+/**
+ * @brief Set the allocation to a preset codec configuration.
+ *
+ * @param codec The preset codec configuration.
+ * @param loc   Location bitmask setting.
+ *
+ */
+static void bt_codec_allocation_set(struct bt_codec_data *codec, enum bt_audio_location loc)
+{
+	codec->data.type = BT_CODEC_CONFIG_LC3_CHAN_ALLOC;
+	sys_put_le32(loc, codec->value);
+	codec->data.data = codec->value;
+	codec->data.data_len = 4;
+}
+
 static int initialize(le_audio_timestamp_cb timestmp_cb)
 {
 	int ret;
 	static bool initialized;
-	struct bt_codec_data bis_codec_data =
-		BT_CODEC_DATA(BT_CODEC_CONFIG_LC3_FREQ, BT_AUDIO_CODEC_CONFIG_FREQ);
+	struct bt_codec_data bis_codec_data[ARRAY_SIZE(audio_streams)];
```

Comment:
Need to discuss. As I can see, the freq parameter is already accessible from level 2.

Reply:
Probably through the preset, but if we wanted to set frequency specifically for one BIS, it would be through here. I know we don't support multiple different frequencies at the time, but we shouldn't remove the code for it, since it would make it easier to change later when we do get support for it. This is particularly important if we want to broadcast one HQ and one SQ stream in the same BIG

Reply:
Will create task on this issue.

Reply:
OCT-2603